spec/models/agency_spec.rb

```diff
@@ -20,8 +12,8 @@
 
   describe ".ordered_agencies" do
     before do
-      Agency.create(name: "Zillow Agency")
-      Agency.create(name: "Acme Agency")
+      create(:agency, name: "Zillow Agency")
+      create(:agency, name: "Acme Agency")
     end
```

Comment:
You can use create_list instead of having to create one agency at a time.  E.g., FactoryGirl.create_list :agency, 3, name: { Faker::Company.name }